jaeger-core/src/main/java/io/jaegertracing/internal/propagation/TextMapCodec.java

```diff
@@ -54,6 +57,25 @@ private TextMapCodec(Builder builder) {
     this.baggagePrefix = builder.baggagePrefix;
   }
 
+  public static JaegerSpanContext contextFromString(String value)
```

Comment:
If it's used only in tests, can't we make this private and test the inject operation instead?

Reply:
This has been blocking the other PR, so I would rather not cause major refactoring of the tests. But going for minimum visibility is always a good goal. In this case, wouldn't all the tests work just as well if this was package level viz?

Reply:
I tried it and sadly no. `propagation` is its own package.

Reply:
Ah, yes. Ok then.

Reply:
I can probably fix the other tests. Don't merge yet.

Reply:
Oops too late. Put another one if a simple fix.